docs/common/lifecycle-message.md

```diff
@@ -0,0 +1,45 @@
+As implied by the name, a `#!typescript LifecycleMessage` refers to some 
+pre-defined system lifecycle, such as the flow of a commit from Git, 
+through CI to deployment and monitoring and so on.
+
+Lifecycle messages are automatically routed to appropriate chat channel by
+the Atomist Bot. Typically, this is the channel automatically created by the
+Atomist Bot when a new project is generated.
+
+Message styling is also currently performed by the Atomist Bot, though we are
+working to allow customization in the handlers themselves.
+
+The Atomist Bot needs two pieces of information in order to correctly route and
+render a lifecycle message, a [GraphNode](/user-guide/rug/path-expressions) and
```

Comment:
I didn't realise - in fact, I'm pretty sure I'm _not_ being notified either way, and this at least keeps the references locally relevant and stops them getting orphaned...